content/api/vision/marker-ids.md

```diff
@@ -0,0 +1,133 @@
+---
+title: Marker IDs
+---
+
+Each marker used in the game has an ID that corresponds to the marker type. You can use this within your code to determine what sort of marker you're looking at and make decisions based off of it.
+
+There are some useful constants which can be imported to do this. For example:
+
+```python
+from robot import WALL, TOKEN_ZONE_1, COLUMN_N_FACING_N
+```
+
+To check your marker matches one of these types, you can use the `in` operator:
+
+```python
+marker, * = r.camera.see()  # Get only the closest marker 
```

Comment:
Note that this is currently invalid (it will `SyntaxError`) as you need to name the variable for the rest of the items.

How about:
``` python
first_marker, *rest = r.camera.see()
```
(which also drops the need for the comment too)

Reply:
(In retrospect I agree that introducing tuple unpacking here may not be the best thing either).

Reply:
I'm going to revert and leave the syntax with indexing. Any benefits of trying to not tell people to index are outweighed by potentially odd looking code that's not directly addressed.